source/dagger/gameplay/team_game/player_controller.cpp

```diff
@@ -10,11 +11,40 @@
 #include "core/graphics/window.h"
 #include "gameplay/common/simple_collisions.h"
 
-#include "core/game/transforms.h"
+
 
 using namespace dagger;
 using namespace lab;
 
+void lab::GenerateRoom(int idNext_,lab::NextLvl& currentLvl_, Transform &tr_)
+{
+    if(currentLvl_.id==0)
+    {
+        auto& view = Engine::Res<Tilemap>()["tilemaps/lab/lab.map"]->tiles;
+        Engine::Registry().destroy(view.begin(),view.end());
```

Comment:
so we are still not cleaning the entire level, why?

Reply:
cleaning the whole registry will have less weird bugs

Reply:
check out racing example, part of code where we restart the game.

Reply:
I thought not to clear the whole level so the player stats (ex health from room to room) don't change. Otherwise we would have to backup everything then put another player entity.